Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -374,7 +374,7 @@ internal override MainLoop Init ()
                                                            );
         }
 
-        CurrentAttribute = new Attribute (ColorName.White, ColorName.Black);
+        CurrentAttribute = new Attribute (ColorName16.White, ColorName16.Black);
```

Comment:
I think you should standardize all this to use the Color instead of Colorname16. If `CursesDriver` supports true color some day, it will be more easy to deal with that.

`CurrentAttribute = new Attribute (Color.White, Color.Black);`

Reply:
I agree, but the change you are looking at happened because I renamed ColorName to ColorName16, not because I directly changed CursesDriver.

I'd rather leave this as-is and have this get addressed in whatever PR fixes https://github.com/gui-cs/Terminal.Gui/issues/2798